portfolio/src/main/java/com/google/sps/data/TimeRange.java

```diff
@@ -26,10 +26,14 @@
   private final Instant start;
   private final Duration duration;
 
-  // Comparator for sorting time ranges by start time
-  public static final Comparator<TimeRange> sortByTimeRangeStartTimeAscending =
+  // Comparator for sorting time ranges  by start time ascending
+  public static final Comparator<TimeRange> SORT_BY_TIME_RANGE_START_TIME_ASCENDING =
       Comparator.comparing(TimeRange::start);
 
+  // Comparator for sorting time ranges by duration ascending and then by start time ascending
+  public static final Comparator<TimeRange> SORT_BY_TIME_RANGE_DURATION_ASCENDING_THEN_START_TIME =
```

Comment:
I'm not sure why this comparator is necessary. I think TimeRangeGroup already keeps the internal list sorted by time window start time. Can we try not to sort again by time window duration?

Reply:
I see. Yes, it's not necessary anymore. It was originally used in testing.

Reply:
After I changed the for loop logic in schedule method, this comparator is not necessary anymore.